src/main/java/org/example/controllers/SalesEmployeeController.java

```diff
@@ -0,0 +1,108 @@
+package org.example.controllers;
+
+import io.swagger.annotations.Api;
+import org.example.exceptions.DoesNotExistException;
+import org.example.exceptions.FailedToCreateException;
+import org.example.models.SalesEmployee;
+import org.example.models.SalesEmployeeRequest;
+import org.example.services.SalesEmployeeService;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.sql.SQLException;
+
+@Api("Sales Employee API")
+@Path("/api/sales-employee")
+public class SalesEmployeeController {
+    private final SalesEmployeeService salesEmployeeService;
+    public SalesEmployeeController(SalesEmployeeService salesEmployeeService) {
+        this.salesEmployeeService = salesEmployeeService;
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getAllSalesEmployees() {
+        try {
+            return Response.ok().entity(
+                    salesEmployeeService.getAllSalesEmployees()
+            ).build();
+        } catch (SQLException e) {
+            return Response.serverError().build();
+        }
+    }
```

Comment:
Could we log exceptions, ditto for other methods. i.e
 ```suggestion
    @GET
    @Produces(MediaType.APPLICATION_JSON)
    public Response getAllSalesEmployees() {
        try {
            return Response.ok().entity(
                    salesEmployeeService.getAllSalesEmployees()
            ).build();
        } catch (SQLException e){
            System.out.println("SQL Exception while trying to list sales employees: " + e);
            return Response.serverError().build();
        }
    }
```

Reply:
will make a ticket to do this for all the files once they're complete, along with the frontend repo! thanks

Reply:
@dylanmckk @CharlotteMKainos @CForde-Kainos fyi